eva-repository/src/main/kotlin/com/razz/eva/repository/JooqEventRepository.kt

```diff
@@ -100,24 +93,12 @@ class JooqEventRepository(
                 constraintName
             )
         }
-
-        val tracingContext = kotlin.runCatching {
-            coroutineContext[ActiveSpanElement]?.span?.context()?.let {
-                mutableMapOf<String, String>().apply {
-                    tracer.inject(it, Format.Builtin.TEXT_MAP, TextMapAdapter(this))
-                }
-            }
-        }.getOrNull()
         val modelEventRs = uowEvent.modelEvents.map { (id, event) ->
             toMERecord(
                 uowEvent = uowEvent,
                 eventId = id,
                 modelEvent = event
-            ).also { record ->
-                if (tracingContext != null) {
-                    record.tracingContext = json.encodeToString(tracingContext)
```

Comment:
Why it is no longer there? How to restore telemetry context from consumer?

Reply:
didn't have time to support consumers